app/views/decidim/action_delegator/admin/import_participants/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_tag(import_participants_path, multipart: true, class: "form new_import") do %>
+  <%= hidden_field_tag :setting_id, current_setting.id %>
+  <div class="card">
+    <div class="card-divider">
+      <h2 class="card-title">
+        <%= t(".title") %>
+      </h2>
+    </div>
+  </div>
+  <p><%= t(".upload_instructions") %></p>
+  <p><%= t(".required_fields", authorization_method: t(".authorization_method.#{current_setting.authorization_method}")) %></p>
+  <p><%= t(".title_example") %></p>
+  <pre class="code-block">
+    email,phone,weight
+    foo@example.org,6660000,1.5
+    bar@example.org,6660001,1
+    baz@example.org,6660002,2
+    ...</pre>
+
```

Comment:
```suggestion
  <pre class="code-block">
    email,phone,weight
    foo@example.org,6660000,1.5
    bar@example.org,6660001,1
    baz@example.org,6660002,2
    ...</pre>
Note that there's also possible to import weights using the name on the ponderation table instead of the weight number, for instance:
  <pre class="code-block">
    email,phone,weight
    foo@example.org,6660000,producer
    bar@example.org,6660001,consumer
    baz@example.org,6660002,associate
    ...</pre>

```

Reply:
Let's add here a little more explanation of the possibilities